### PR TITLE
V4.0.1-st (STABLE RELEASE)

### PR DESCRIPTION
**Pull request by Gilbert142**
Dear reviewers and contributors, please clone this branch (release-4.0.1) locally, and follow the self-hosting guide on this repo's wiki to set it up. MAKE SURE EVERYTHING WORKS.

**This will be your last chance to fix bugs before we push to stable!**

Config.json will be updated to match the correct branch. No other changes will be pushed apart from bug fixes.

### CODE REVIEW

**Comment by KingDGrizzle** [comment]
I'll clone this locally, and review accordingly

**Comment by KingDGrizzle** [comment]
Hey @HilbertGilbertson wanna fix lottery?

**Review by vladfrangu — CHANGES_REQUESTED** [review]
Oh, maybe i should actually init this? serverData.name in maintainer page?

File: Web/views/pages/maintainer-version.ejs [hunk 1]
```diff
@@ -0,0 +1,146 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title><%= serverData.name %> Maintainer Console</title>
```
Comment (GilbertGobbels) [inline Web/views/pages/maintainer-version.ejs]: Returns the bot name, AFAIK

**Review by vladfrangu — DISMISSED** [review]

**Review by vladfrangu — APPROVED** [review]
Unless you are going to do any more changes, which won't be very hard to issue an update from the maintainer panel, I give my approval. Finishing touches should be implemented, and merged. Have fun GAB users!

**Pull request merged**